app/views/courses/EnrollmentsView.coffee

```diff
@@ -0,0 +1,133 @@
+app = require 'core/application'
+CreateAccountModal = require 'views/core/CreateAccountModal'
+Classroom = require 'models/Classroom'
+CocoCollection = require 'collections/CocoCollection'
+Course = require 'models/Course'
+Prepaids = require 'collections/Prepaids'
+RootView = require 'views/core/RootView'
+stripeHandler = require 'core/services/stripe'
+template = require 'templates/courses/enrollments-view'
+User = require 'models/User'
+utils = require 'core/utils'
+Products = require 'collections/Products'
+
+module.exports = class EnrollmentsView extends RootView
+  id: 'enrollments-view'
+  template: template
+  numberOfStudents: 30
+  pricePerStudent: 0
+
+  initialize: (options) ->
+    @listenTo stripeHandler, 'received-token', @onStripeReceivedToken
+    @fromClassroom = utils.getQueryVariable('from-classroom')
+    @members = new CocoCollection([], { model: User })
+    @listenTo @members, 'sync', @membersSync
+    @classrooms = new CocoCollection([], { url: "/db/classroom", model: Classroom })
+    @classrooms.comparator = '_id'
+    @listenToOnce @classrooms, 'sync', @onceClassroomsSync
+    @supermodel.loadCollection(@classrooms, 'classrooms', {data: {ownerID: me.id}})
+    @prepaids = new Prepaids()
+    @prepaids.comparator = '_id'
+    @prepaids.fetchByCreator(me.id)
+    @supermodel.loadCollection(@prepaids, 'prepaids')
+    @products = new Products()
+    @supermodel.loadCollection(@products, 'products')
+    super(options)
+
+  events:
+    'input #students-input': 'onInputStudentsInput'
+    'click .purchase-now': 'onClickPurchaseButton'
+    # 'click .enroll-students': 'onClickEnrollStudents'
+
+  onLoaded: ->
+    @pricePerStudent = @products.findWhere({name: 'course'}).get('amount')
+    me.setRole 'teacher'
+    super()
+
+  getPriceString: -> '$' + (@getPrice()/100).toFixed(2)
+  getPrice: -> @pricePerStudent * @numberOfStudents
+
+  onceClassroomsSync: ->
+    for classroom in @classrooms.models
+      @members.fetch({
+        remove: false
+        url: "/db/classroom/#{classroom.id}/members"
+      })
+
+  membersSync: ->
+    @memberEnrolledMap = {}
+    for user in @members.models
+      @memberEnrolledMap[user.id] = user.get('coursePrepaidID')?
+    @classroomNotEnrolledMap = {}
+    @totalNotEnrolled = 0
+    for classroom in @classrooms.models
+      @classroomNotEnrolledMap[classroom.id] = 0
+      for memberID in classroom.get('members')
+        @classroomNotEnrolledMap[classroom.id]++ unless @memberEnrolledMap[memberID]
+      @totalNotEnrolled += @classroomNotEnrolledMap[classroom.id]
+    @numberOfStudents = @totalNotEnrolled
+    @render?()
+
+  onInputStudentsInput: ->
+    @numberOfStudents = Math.max(parseInt(@$('#students-input').val()) or 0, 0)
+    @updatePrice()
+
+  updatePrice: ->
+    @renderSelectors '#price-form-group'
+
+  numberOfStudentsIsValid: -> @numberOfStudents > 0 and @numberOfStudents < 100000
```

Comment:
By the way, CoffeeScript has `0 < @numberOfStudents < 10000`, in case you didn't know.